### ISSUE TITLE
README.md missing in v0.6

### ISSUE THREAD

**Issue by chid** [django-silk/silk#125]
The current version in pypi has an issue with a missing file (README.md), it does not appear to be within the package.

```
  Using cached django-silk-0.6.0.tar.gz
    Complete output from command python setup.py egg_info:
    warning: pypandoc module not found, could not convert Markdown to RST
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/pip-build-s0wzt75r/django-silk/setup.py", line 13, in <module>
        README = read_md('README.md')
      File "/tmp/pip-build-s0wzt75r/django-silk/setup.py", line 11, in read_md
        def read_md(f): return open(f, 'r').read()
    FileNotFoundError: [Errno 2] No such file or directory: 'README.md'
```

**Comment by avelis** [comment]
I made this change so I could convert markdown files to rst when pushing a release to pypi. This reduces the need to manage two readme files. My mistep was assuming the setup was including the markdown version. I will update the package includes to include that file.

**Comment by avelis** [comment]
Addressed with PR #128
